contrib/builder/deb/aarch64/generate.sh

```diff
@@ -83,12 +83,20 @@ for version in "${versions[@]}"; do
 			;;
 	esac
 
+    case "$suite" in
+        jessie)
+            echo 'RUN echo deb http://ftp.debian.org/debian jessie-backports main > /etc/apt/sources.list.d/backports.list' >> "$version/Dockerfile"
+            ;;
+        *)
+            ;;
+    esac
```

Comment:
ok. adding ```libseccomp``` support for jessie is rather straightforward, and I have that change done (just need to reverify). I'd rather add that than completely removing jessie.